department-of-veterans-affairs/caseflow
# Feature/appeals 17521/legacy edit issues

Resolves [17521](https://vajira.max.gov/browse/APPEALS-17521)

# Description
Opening up Intake's Edit Issue's page for legacy appeals. The only functionality this will be used for is to update an existing issue's MST or PACT status. All other functionalities in this page will be removed for legacy appeals.

## Acceptance Criteria
- [ ] When the user selects "Correct Issues" for a legacy appeal on Case Details, the Edit Issue page displays.
- [ ] The user has access to the Edit Issue dropdown for BA Intake Admin (Supervisors) and Clerks of the Board (COB)
- [ ] The user can select "Edit Issue" from the dropdown and be taken to the Edit Issue modal

## Testing Plan
1. Login as BVA Intake Admin or Clerk of the Board User
2. Search for any Legacy Appeal that has at least one issue then click on the docket number.
3. On the Case Details Page, click on "Correct Issues" labeled next to the Issues section
4. Observe that you are taken to the Edit Issues page.
5. Observe that there is no "+ Add Issue" button on the page.
6. Click on "Select action"
7. Observe that the only option available in the dropdown is "Edit Issue"
8. Click on "Edit Issue"
9. Observe that the Edit Issue modal appears.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
